### PR TITLE
Fix bug with lbfPlutusRust not generating Json and Eq instances

### PR DESCRIPTION
**Pull request by szg251**

### CODE REVIEW

File: testsuites/lbt-plutus/lbt-plutus-typescript/.envrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+nix_direnv_watch_file build.json
+use flake ../../..#lbt-plutus-typescript
```
Comment (bladyjoker) [inline testsuites/lbt-plutus/lbt-plutus-typescript/.envrc:2]: Hmm i don't think that's a dev shell, should be dev-lbt-plutus-typescript or smtn?

File: testsuites/lbt-prelude/lbt-prelude-typescript/.envrc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+nix_direnv_watch_file build.json
+use flake ../../..#lbt-prelude-typescript
```
Comment (bladyjoker) [inline testsuites/lbt-prelude/lbt-prelude-typescript/.envrc:2]: See above comment!

File: testsuites/lbt-prelude/lbt-prelude-typescript/.envrc [hunk 3]
```diff
@@ -0,0 +1,2 @@
+nix_direnv_watch_file build.json
```
Comment (bladyjoker) [inline testsuites/lbt-prelude/lbt-prelude-typescript/.envrc:1]: This command is not available in new direnv for nix. What's build.json btw?

**Review by bladyjoker — APPROVED** [review]
Thanks! Check the comments:)

File: testsuites/lbt-prelude/lbt-prelude-typescript/.envrc (hunk 3 above)
Comment (szg251) [reply testsuites/lbt-prelude/lbt-prelude-typescript/.envrc:1]: Oh, I didn't know
I'll remove these

File: testsuites/lbt-plutus/lbt-plutus-typescript/.envrc (hunk 1 above)
Comment (szg251) [reply testsuites/lbt-plutus/lbt-plutus-typescript/.envrc:1]: This should be solved in flake-lang first, created an issue: https://github.com/mlabs-haskell/flake-lang.nix/issues/39

**Pull request merged**